tests/unit/cycling/test_iso8601.py

```diff
@@ -892,10 +895,10 @@ def test_next_simple_no_now(set_cycling_type):
     point = "next(T00Z)+P1D"
     output = ingest_time(point, my_now)
 
-    current_time = datetime.utcnow()
+    current_time = datetime.now(timezone.utc)
     # my_now is None, but ingest_time will have used a similar time, and
     # the returned value must be after current_time
-    output_time = datetime.strptime(output, "%Y%m%dT%H%MZ")
+    output_time = datetime.strptime(output, "%Y%m%dT%H%M%z")
```

Comment:
Time zone format has changed?

Reply:
Using `Z` just means it matches and discards the `Z` at the end of `output`, leading to a naive `datetime`. Whereas using `%z` means it recognises `Z` as UTC, leading to a tz-aware `timezone`. The latter is needed for the comparison operation

Reply:
What I'm asking is why is this diff present? The test passed as was before these changes.

Is this arbitrary "tidying", or was it needed due to a change in timestamp format?